include/sway/config.h

```diff
@@ -160,6 +166,7 @@ struct output_config {
 	int x, y;
 	float scale;
 	int32_t transform;
+	enum config_transform_type transform_type;
```

Comment:
I really think there's no need for this field. We can use `wlr_output_transform_compose` in `output_cmd_transform`.

Reply:
I'm all for simplifying, but am not sure how then you'd signal that the transform should be relative rather than absolute.

Reply:
1. Get the current output transform
2. `wlr_output_transform_compose(current, relative)`

Reply:
Sure, but are you proposing doing that in `output/transform.c`?

Reply:
Yes

Reply:
ok, is `oc->name` set at that point? I assume I'd have to pull in `tree/output` and parse as `apply_output_config` does?

Reply:
Yes, `oc->name` is set (you can `sway_assert` it). You can use `output_by_name` to get the `sway_output`.

Reply:
Sounds good, thanks.